doc/develop/application/index.rst

```diff
@@ -45,6 +45,7 @@ Here are the files in a simple Zephyr application:
    ├── CMakeLists.txt
    ├── app.overlay
    ├── prj.conf
+   ├── VERSION
    └── src
        └── main.c
```

Comment:
I was about to suggest the following change 

```suggestion
   └── src
       └── main.c
   └── VERSION
```

but then realized that CMakeLists.txt is also not properly sorted, so I guess I can live with that, ...or ask if you'd be willing to add a commit sorting the files/folders alphabetically and using the same order for the bullet list that follows?

Reply:
I would keep the current list order, as it reflects the development order: CMakeLists.txt points to application files, the app.overlay application file defines hardware configuration, the prj.conf defines the software configuration. So the current order makes sense.

Reply:
Ya but I find the fact it doesn't correspond to the way it would come out of `ls` confusing... Anyway, probably nitpicking.